l10n/de/suite/chrome/common/pref/pref-cache.dtd

```diff
@@ -7,7 +7,7 @@
 <!--LOCALIZATION NOTE : FILE Cache prefs dialog -->
 <!ENTITY pref.cache.title                "Cache">
 <!ENTITY pref.cache.caption              "Cache-Optionen setzen">
-<!ENTITY cachePara                       "Der Cache behält Kopien häufig besuchter Webseiten auf Ihrer Festplatte. (Wenn Sie &quot;Neu laden&quot; klicken, wird immer die aktuellste Version einer Seite angezeigt.)">
+<!ENTITY cachePara                       "Der Cache behält Kopien häufig besuchter Webseiten auf deiner Festplatte. (Wenn du &quot;Neu lädst&quot; klickst, wird immer die aktuellste Version einer Seite angezeigt.)">
```

Comment:
I think the expression "Neu laden" should be left as is. It seems to be some menu item name quotation, not an imperative.